src/python/pants/backend/python/lint/python_format_target_integration_test.py

```diff
@@ -81,3 +79,14 @@ def test_multiple_formatters_changing_different_files(self) -> None:
         ]
         results = self.run_black_and_isort(original_sources)
         assert results.combined_digest == self.get_digest(fixed_sources)
+
+    def test_skipped_formatter(self) -> None:
```

Comment:
I checked that this test will fail if we don't have the corresponding fix in `python_format_target.py`.